x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/middleware.ts

```diff
@@ -233,6 +248,27 @@ export const endpointMiddlewareFactory: ImmutableMiddlewareFactory<EndpointState
         });
       }
     }
+
+    if (action.type === 'serverToggledEndpointListAutoRefresh' && isAutoRefreshEnabled(state)) {
+      startPoll({
+        pollAction: () => {
+          dispatch({
+            type: 'appRequestedEndpointList',
+          });
+        },
+        interval: POLL_INTERVAL,
+        shouldStop: () => {
+          const currentState = getState();
```

Comment:
honest question, since I don't know what state this refers to:

is this an expensive call, in any way? `getState()`? Since it's in the hot path of being called every interval

Reply:
This call looks up the data we have in state (aka memory in the browser), so it's not too expensive, just a data store in the FE.  It's not a server call or anything